core/runtime/src/main/java/org/apache/isis/core/runtime/headless/logging/LeveledLogger.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.isis.core.runtime.headless.logging;
```

Comment:
just to let you know: files do need a license header; but I can fix that after merge